tests/test_bessel.py

```diff
@@ -374,6 +374,133 @@ def test_jv_root():
     with assert_raises(RuntimeError):
         galsim.bessel.jv_root(0.3,-1)
 
+@timer
+def test_gammainc():
+    """Test the bessel.gammainc function"""
+    a_list = [ 3.3, 4, 1.9, 0, 0.2, 4.7, 39.8, 2.3 ]
+    x_list = [ 1, 2.01, 0.2, 3.3, 15.9, 1.e-12, 3.4, 19.1 ]
+    vals1 = [ galsim.bessel.gammainc(a,x) for a,x in zip(a_list,x_list) ]
+    print('x = ',x_list)
+    print('vals1 = ',vals1)
+
+    vals2 = [ scipy.special.gammainc(a,x) for a,x in zip(a_list,x_list) ]
+    print('vals2 = ',vals2)
+    np.testing.assert_allclose(
+        vals1, vals2, rtol=1.e-10, err_msg="bessel.gammainc disagrees with scipy.special.gammainc")
+
+    # These values are what scipy returns.
+    vals2 = [   0.05336162361853338,
```

Comment:
I do not fully understand the logic in testing twice against scipy, once calling the relevant routine and once using stored values.  It's not *wrong* but it seems redundant.  (Comment applies to some other tests.)  Is there some reason that I am missing?

Reply:
It is redundant.  It used to be useful when scipy was optional (since it used to be a pain to install), but we now require it for tests.  I still find it slightly useful to scan over the values to make sure we're hitting a good dynamic range of possible outputs as well as inputs, so I haven't gotten rid of them.